playwright-test/atb.spec.ts

```diff
@@ -0,0 +1,110 @@
+import { test, expect, mockAtb } from "./harness";
+
```

Comment:
I think we should rethink the directory structure a bit, so that it's clear that the playwright tests are integration tests, so that it's clear that the old integration tests are legacy ones, and so that it's clear which shared files (data and utilities) are shared between the Playwright and legacy integration tests.

Ideally this would be explained in a README or similar too, so folks writing new tests no where to put them and which examples to copy. Otherwise, new tests will be written by copying the legacy tests and we'll have more work to do if we want to eventually migrate them.

Reply:
How about we move playwright tests to the root of the `integration-test` folder, then put the old tests in a `legacy` subfolder of that?

Reply:
Sounds good to me :+1: